native_client/tflitemodelstate.cc

```diff
@@ -90,6 +99,61 @@ TFLiteModelState::~TFLiteModelState()
 {
 }
 
+std::map<std::string, tflite::Interpreter::TfLiteDelegatePtr>
+TFLiteModelState::get_delegates()
+{
+  std::map<std::string, tflite::Interpreter::TfLiteDelegatePtr> delegates;
+
+  std::string env_delegate(std::getenv("DS_TFLITE_DELEGATE"));
+
+#ifdef __ANDROID__
+  if (env_delegate == std::string("gpu")) {
+    LOGE("Trying to get GPU delegate ...");
```

Comment:
Why the error log?

Reply:
And same question for the log calls below. These look like debug messages.

Reply:
Android logcat is sometimes funky, and might not always be consistent.